### ISSUE TITLE
Does Ethereum Wallet Have a Pneumonic Phrase/Seed to recover my wallet, if so where?

### ISSUE THREAD

**Issue by tkellgren** [ethereum/mist#3223]
I have both mist and Ethereum wallet, both updated to the newest versions.

I was writing in my notebook all my different wallet's recovery phrases, then I got to my etherum wallet.

IS there a way to recover via phrase/seed like electrum for example, or should I save the key  UTC file or w/e to back it up?

Thanks guys, much love and even more Hash!

**Comment by jontis** [comment]
I'm pretty sure that there's not, they're still having more severe bugs to fix. It would be a great security and convenience feature later.

**Comment by achempion** [comment]
You can write content of your mist wallet file. It contains json with your wallet data: address, encoded secret and algorithm to decode private key with your password from that secret. Also you can decode private key by yourself and write it down.

**Comment by tkellgren** [comment]
Yeah I was going to reply I have the utc file to recover my wallet on
myetherwallet if I need to Thanks again

On Oct 29, 2017 10:50 AM, "Boris Kuznetsov" <notifications@github.com>
wrote:

You can write content of your mist wallet file. It contains json with your
wallet data: address, encoded secret and algorithm to decode private key
with your password from that secret. Also you can decode private key by
yourself and write it down.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/ethereum/mist/issues/3223#issuecomment-340272105>, or mute
the thread

**Comment by Fiquel** [comment]
Forgive my lack of knowledge, but I have a text file, with a name that starts UTC....., in Windows Explorer the file type is listed as random numbers and letters. Can I use this file to access my account as I do not remember having ever set up a password when I first set up my Mist account.

Thanks

**Comment by tkellgren** [comment]
Your UTC file is also called keystore on myetherwallet.com, you can go-to
view wallet info for example, choose keystore and pick the UTC file.  Most
likely it's encrypted and requires a password but worth a try.

**Comment by Fiquel** [comment]
Thanks, and you're correct, it requires the password, and I'm 97% sure I was not prompted to set one up. Any ideas of my options?

**Comment by tkellgren** [comment]
Not off the top of my head but I'm by no means an expert

**Comment by Fiquel** [comment]
It seems to be a recurring issue. Thanks

**Comment by kooweele** [comment]
Looks like Mist is an Ether Death-trap.

**Comment by jontis** [comment]
Always back up keys and try restoring accounts from those files before you load any crypto into those accounts.

**Comment by kooweele** [comment]
Don’t understand ... is Mist(Etherbase) account a valid one that I have the password on?!
This Mist wallet has lack of documentation and a lot of ppl are now having their Ether trapped in it with the correct password.
 

Sent from Yahoo Mail for iPhone


On Sunday, December 10, 2017, 18:08, jontis <notifications@github.com> wrote:


Always back up keys and try restoring accounts from those files before you load any crypto into those accounts.

**Comment by jontis** [comment]
Never trust things. Always send in a small amount that you don't care about losing and try to send it out again before you send in larger amounts. 101 with all types of money, banks crypto and similar.

**Comment by kooweele** [comment]
Yeah ... lessons learnt ... this Mist Wallet is an Ether black hole. 


Sent from Yahoo Mail for iPhone


On Sunday, December 10, 2017, 20:23, jontis <notifications@github.com> wrote:


Never trust things. Always send in a small amount that you don't care about losing and try to send it out again before you send in larger amounts. 101 with all types of money, banks crypto and similar.

**Comment by jontis** [comment]
It is a very BETA software yes. There is a reason normal people are just vaguely beginning to grasp BTC. They are about 5 years from Ether.

**Comment by kooweele** [comment]
Looks like commercial solution like ledger nano is the way to go then these Github open source ones. 


Sent from Yahoo Mail for iPhone


On Sunday, December 10, 2017, 20:37, jontis <notifications@github.com> wrote:


It is a very BETA software yes. There is a reason normal people are just vaguely beginning to grasp BTC. They are about 5 years from Ether.

**Comment by jontis** [comment]
I like the idea of having a spare key to a access a larger investment if something fucks up.

**Comment by kooweele** [comment]
A lot of websites and YouTube videos are recommending Mist as “top 3 Ether” wallets ... this is completely crap and unprofessional of them. 


Sent from Yahoo Mail for iPhone


On Sunday, December 10, 2017, 20:47, jontis <notifications@github.com> wrote:


I like the idea of having a spare key to a access a larger investment if something fucks up.

**Comment by jontis** [comment]
Maybe all the others are even worse?

**Comment by rustyx** [comment]
Pneumonic :) That's one really sick seed :)
The word you were looking for is Mnemonic.
